superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -489,6 +531,7 @@ void PrintResultTag(const BenchArgs &args) {
 int RunCopy(BenchArgs *args) {
     cudaError_t cuda_err = cudaSuccess;
     uint64_t num_thread_blocks;
+    const uint64_t num_warm_up = 20;
```

Comment:
use a larger warmup number? why not support it as an argument

Reply:
Still trying best warmup number. Made it an option.